ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -161,16 +161,17 @@ public bool HasUpToDateBundleInCache()
             if (_isDevSupportEnabled)
             {
 #if WINDOWS_UWP
-                var lastUpdateTime = Windows.ApplicationModel.Package.Current.InstalledDate.UtcDateTime;
+                var lastNativeUpdateTime = Windows.ApplicationModel.Package.Current.InstalledDate.UtcDateTime;
                 var localFolder = ApplicationData.Current.LocalFolder.Path;
 #else
-                var lastUpdateTime = File.GetCreationTime(Assembly.GetExecutingAssembly().Location);
+                var lastNativeUpdateTime = File.GetLastWriteTimeUtc(Assembly.GetEntryAssembly().Location);
```

Comment:
Taking the LastWriteTime instead of the CreationTime.   The CreationTime does not change between builds even when there are native changes.   

Timestamp was being taken with local time where as the rest of the method is using UTC.   Changed code to take UTC